srcpkgs/mongroup/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'mongroup'
 pkgname=mongroup
 version=0.4.1
-revision=1
+revision=2
+noarch=yes
+depends="mon"
 short_desc="Monitor a group of processes with mon"
 maintainer="Diogo Leal <diogo@diogoleal.com>"
 license="GPL-3"
```

Comment:
This one doesn't actually have a license listed.

So it is all rights reserved and propietary.

Arch Linux uses MIT for it but not MIT license is present.

@the-maldridge @Vaelatern